lib/testing/src/equality/equality.dart

```diff
@@ -94,7 +94,7 @@ class _EqualityGroupMatcher extends Matcher {
     });
 
     // Run the test multiple times to ensure deterministic equals
-    for (var run in range(_repetitions)) {
```

Comment:
Analyzer is complaining. Creating a new iterable doesn't seem like something we should model/encourage folks to do